package-lock.json

```diff
@@ -1,28 +1,8182 @@
 {
```

Comment:
please always use `npm ci` instead of `npm install`, this way it installs the versions of the package.json.lock, which should be our single source of truth